src/dialogs/PaymentErrorDialog.vue

```diff
@@ -29,8 +29,8 @@
         <div class="font-15 mb-4" v-if="numErrors > 0">
           <p>We were unable to process your payment due to the following errors:</p>
           <ul>
-            <li v-for="(error, index) in errors" :key="index">
-              {{error.message}}
+            <li v-for="(error, index) in formatedError" :key="index">
+              <span v-html="error" id="formated-error-msg"></span>
             </li>
```

Comment:
Local variable after transformation.

Reply:
Why do you need the span? The v-html can be right in the List Item element.

This is probably simpler if you change the Unordered List / List Items to something else.